.github/shared/src/readme.js

```diff
@@ -12,6 +12,11 @@ import { Tag } from "./tag.js";
  * @typedef {import('./spec-model.js').ToJSONOptions} ToJSONOptions
  */
 
+/**
+ * Regex to match tag names in readme.md yaml code blocks
+ */
+export const TagMatchRegex = /yaml.*\$\(tag\) ?== ?["']([^'"]*?)["']/;
```

Comment:
I suspect we don't care but should we match the same quote on each side? as opposed to something like `"abc'`?